cms/templates/course_outline.html

```diff
@@ -100,6 +100,23 @@ <h2 class="status-release-label">${_("Course Start Date:")}</h2>
                       </li>
                     </ul>
                 </div>
+                % if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False) and license:
```

Comment:
Its the course content license, its passed to the view just like course_release_date. Should this be changed to course_content_license?